src/styles/themer/withTheme.js

```diff
@@ -0,0 +1,41 @@
+// Higher order component that force updates component on themer change
+// and passes down theme through props
+import React, { Component } from 'react'
+import themer from './index'
+import { themePropTypes } from './utils'
+
+function withTheme(InnerComponent) {
+  class Wrapped extends Component {
+    static displayName = `withTheme(${InnerComponent.displayName})`
+
+    static propTypes = {
+      snacksTheme: themePropTypes
+    }
+
+    componentDidMount() {
+      this.unsubscribe = themer.subscribe(this.onThemeChange)
+    }
+
+    componentWillUnmount() {
+      this.unsubscribe()
+    }
+
+    onThemeChange() {
```

Comment:
seems like this may need to be defined as an attribute i.e `onThemeChange = () =>` since it's being passed as a callback